internal/cli/cmd/encrypt.go

```diff
@@ -57,14 +58,30 @@ func newCmdIPsecRotateKey() *cobra.Command {
 		Long:  "This command rotates IPsec encryption key in the cluster",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			params.CiliumNamespace = namespace
+			if err := checkParams(params); err != nil {
+				fatalf("Input params are invalid: %s", err)
+			}
 			s := encrypt.NewEncrypt(k8sClient, params)
 			if err := s.IPsecRotateKey(context.Background()); err != nil {
 				fatalf("Unable to rotate IPsec key: %s", err)
 			}
 			return nil
 		},
 	}
+	cmd.Flags().StringVarP(&params.IPsecKeyAuthAlgo, "auth-algo", "", "", "IPsec key authentication algorithm. One of: gcm-aes, hmac-md5, hmac-sha1, hmac-sha256, hmac-sha512")
+	cmd.Flags().StringVarP(&params.IPsecKeyPerNode, "key-per-node", "", "", "IPsec key per cluster node. One of: true, false")
```

Comment:
Having gone through the changes above, I believe empty values are also allowed for `--auth-algo` and `--key-per-node`: users don't have to specify these two flags if want to keep the same auth-algo and per-node settings as existing ones. Maybe worth mentioning in flag description that they are optional flags and empty values mean using the current settings.

Reply:
Makes sense! Added, thank you!